src/WordPress/PluginActions.php

```diff
@@ -87,7 +87,7 @@ public function applyDefaultSettings()
             throw new ZoneSettingFailException();
         }
 
-        $currentPlan = $details['result']['plan']['legacy_id'];
+        $currentPlan = $details['result']['plan']['legacy_id'] ?? 'free';
```

Comment:
this is the result of fixing 

```
1) CF\WordPress\Test\PluginActionsTest::testReturnApplyDefaultSettingsChangeZoneSettingsThrowsZoneSettingFailException
Trying to access array offset on value of type bool

/app/src/WordPress/PluginActions.php:90
/app/src/Test/WordPress/PluginActionsTest.php:121
```

Prior to this change, it was returning `false` (as I assume one of the keys weren't present) however I updated it to default to "free" which should cover all of our use cases and a safe assumption given free accounts can do basic things.